components/src/tabs/tab-item.component.ts

```diff
@@ -0,0 +1,56 @@
+import { Component, Input } from "@angular/core";
+
+@Component({
+  selector: "bit-tab-item",
+  templateUrl: "./tab-item.component.html",
+})
```

Comment:
Another variant would be to make this a directive. Which would let you use either `a` or `button` depending on the context.